Content/Onboarding/onboarding.json

```diff
@@ -13,5 +13,15 @@
 
 	"onboarding_eula_headerTitle": "User Terms",
 	"onboarding_eula_headerSubtitle": "To proceed, you must accept the user terms below.",
-	"onboarding_eula_accept": "Accept"
+	"onboarding_eula_accept": "Accept",
+
+	"onboarding_android_cloud_backupTitle": "Back up your wallet settings",
```

Comment:
We should aim to have max 3 levels deep, so maybe rename to `onboarding_androidCloud_xyz`?

Reply:
Maybe `onboarding_android_cloudBackupTitle`?

Reply:
I am happy to say that you are both wrong, `onboarding_cloudAndroid_xyz` is the real answer...

Reply:
thank you guys! 😄 

I updated them accordingly.